hack/package-helm-charts.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o pipefail
+
+# if arg1 is set, it will be used as the version number
+if [ -z "$1" ]; then
+  VERSION=$(awk -F= '/^VERSION/ { print $2 }' versions.mk | tr -d '[:space:]')
+else
+  VERSION=$1
+fi
+VERSION=${VERSION#v}
+
+# Create release assets to be uploaded
+helm package deployments/helm/nvidia-dra-driver-gpu/ --version $VERSION --app-version $VERSION
```

Comment:
I am not sure if going `nvidia-dra-driver-gpu` or `k8s-dra-driver-gpu`

Reply:
What do you mean? For the current source code the path you specified here is correct. Are you asking whether this should be changed?

Reply:
yeah the name, what do we prefer? this is me wondering about naming, not a blocker

Reply:
What is the name of the chart that we use in the documentation that we have? How does this compare to the device plugin and the GPU operator?

Reply:
`nvidia-dra-driver-gpu`